openlibrary/plugins/openlibrary/js/image_change.js

```diff
@@ -0,0 +1,77 @@
+export function initChangeImageImport() {
+
+    // Pull data from data-config of class "manageCovers" in covers/manage.html
+    var data_array = $('.manageCovers').attr('data-config').split(',');
```

Comment:
No need to do split here, we could do JSON.parse instead.
Also use const rather than var.